src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const INPUT_SYMBOL = fromFormat.pop();
+  const OUTPUT_SYMBOL = toFormat.pop();
+
+  const INPUT_DATE_ARRAY = date.split(INPUT_SYMBOL);
+
+  const inputDate = {
+    day: INPUT_DATE_ARRAY[fromFormat.indexOf('DD')],
+    month: INPUT_DATE_ARRAY[fromFormat.indexOf('MM')],
+    year:
+      INPUT_DATE_ARRAY[fromFormat.indexOf('YYYY')] ||
+      INPUT_DATE_ARRAY[fromFormat.indexOf('YY')],
```

Comment:
There is a potential issue with the way you are handling years. If the `fromFormat` array contains both 'YYYY' and 'YY', your code will always take the value associated with whichever comes last in the `fromFormat` array. You should explicitly check for both 'YYYY' and 'YY' and handle them accordingly.